packages/extension-ui/src/Popup/Signing/LedgerSign.tsx

```diff
@@ -69,43 +74,56 @@ function LedgerSign ({ accountIndex, addressOffset, className, error, genesisHas
 
   const _onSignLedger = useCallback(
     (): void => {
-      if (!ledger || !payload || !onSignature || !chain) {
+      if (!ledger || !payloadJson || !onSignature || !chain || !payloadExt) {
```

Comment:
Everything is working - up until "Sign on ledger" - "chain" is coming back null - but this `genesisHash` is there. This means the issue starts with `getMetadata`.

Reply:
Okay cool, it works when I update the metadata!